### PR TITLE
CNN using Pytorch

### PR DESCRIPTION
**Pull request by ayanmaity201**
@aribis369 , I have made a notebook , **Pytorch_CNN.ipynb** ,where I implemented CNN using pytorch torch.nn module on MNIST data. I added that notebook in examples-pytorch folder . Please merge my Pull Request.

### CODE REVIEW

**Comment by aribis369** [comment]
@ayanmaity201 Can you add a classification report using scikit at the end to get class-wise results.

**Comment by ayanmaity201** [comment]
@aribis369 , CNN was taking too much time to get trained so, I did not show any accuracy score at the end.

**Comment by ayanmaity201** [comment]
@aribis369 There is no module for CNN in scikit learn . As such deepNN requires GPU supports which is not tiil now in sklearn, there is no ready-to-use module for CNN in sklearn.  So, It is not possible to compare the pytorch model with sklearn. Also I have updated my notebook **Pytorch_CNN_updated.ipynb** by adding a accuracy measure for the model  on a test data.

**Comment by aribis369** [comment]
@ayanmaity201 I asked for classification report for each class for which you can use scikit for easy implementation, you train your models predict results in test set and then you can show a classification report. First of all search for classification report in scikit.

**Comment by ayanmaity201** [comment]
I am still not getting what you want to say. What do you mean by " search for classification report in scikit ". Please elaborate the whole thing that you want.

**Comment by aribis369** [comment]
@ayanmaity201 See the mnist program in example-keras folder of this repo.

**Comment by ayanmaity201** [comment]
@aribis369 , Now I get what you are trying to say. Thanks and  I will update the notebook very soon.

**Comment by ayanmaity201** [comment]
@aribis369 , I have given the classification report at the end of the notebook and updated the notebook to **Pytorch_CNN_Tutorial.ipynb** in examples-pytorch folder. Now you can merge it.

**Comment by aribis369** [comment]
@ayanmaity201 Nice work.

**Comment by aribis369** [comment]
Merged.
